### PR TITLE
Match discover tinder like swipe

### PR DESCRIPTION
**Pull request by tomaszferens**
Screen Recording 2020-11-01 at 01 27 11 PM

### CODE REVIEW

File: sweet.up/DiscoverAndMatch/categories.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+struct Category: Identifiable {
+  var id = UUID()
+
+  var name: String
+  var textOnImage: String
+  var description: String? = nil
+}
+
+let categories: [String: Category] = [
```
Comment (marckraw) [inline sweet.up/DiscoverAndMatch/categories.swift:18]: So this constant is automatically accessed in whole project ?  :D

File: sweet.up/DiscoverAndMatch/categories.swift [hunk 2]
```diff
@@ -0,0 +1,31 @@
+//
+//  categories.swift
+//  sweet.up
+//
+//  Created by Tomasz Ferens on 25/10/2020.
+//
+
+import Foundation
+
+struct Category: Identifiable {
```
Comment (marckraw) [inline sweet.up/DiscoverAndMatch/categories.swift:10]: So this struct is automatically accessed in whole project ? :D

**Review by marckraw — APPROVED** [review]

File: sweet.up/DiscoverAndMatch/categories.swift (hunk 2 above)
Comment (tomaszferens) [reply sweet.up/DiscoverAndMatch/categories.swift:10]: yes :D

File: sweet.up/DiscoverAndMatch/categories.swift (hunk 1 above)
Comment (tomaszferens) [reply sweet.up/DiscoverAndMatch/categories.swift:18]: yes :D

**Pull request merged**